lib/src/test/java/com/uber/cherami/client/CheramiIntegrationTest.java

```diff
@@ -1,302 +0,0 @@
-/*******************************************************************************
```

Comment:
The existing integ tests are simply broken and they aren't currently run as part of the regular builds. And I have a work in progress that totally rewrites the integration test (along with a proper implementation of cherami-server mock). When that patch is out, this file will be removed anyways. Given that background, this patch simply deletes the file instead of attempting to fix the references within that file.